src/pretix/control/views/item.py

```diff
@@ -746,9 +727,12 @@ def get_answer_statistics(self):
 
     def get_context_data(self, **kwargs):
         ctx = super().get_context_data()
-        ctx['items'] = self.object.items.all()
+        ctx['items'] = self.object.items.exists()
+        ctx['has_subevents'] = self.request.event.has_subevents
         stats = self.get_answer_statistics()
-        ctx['stats'], ctx['total'] = stats
+        if stats:
+            ctx['stats'], ctx['total'] = stats
```

Comment:
`stats` always evaluates to True even if result is empty `([], 0)`
```suggestion
        ctx['stats'], ctx['total'] = stats
```